config/locales/en.yml

```diff
@@ -32,3 +35,21 @@ en:
         setting: "Settings"
         logout: "Log out"
         login: "Log in"
+  users:
```

Comment:
Duplicate namespace. Please the translation into the previously declared users section